.github/workflows/release.yml

```diff
@@ -151,10 +151,13 @@ jobs:
         include:
           - name: servicecontrol
             project: ServiceControl
+            description: ServiceControl primary instance
```

Comment:
I know we call this "primary" in the code, but in docs, we seem to either call this an "error" instance or just a "ServiceControl instance". 

Should we align the description here with what we call it in docs?